src/core_plugins/kibana/public/visualize/editor/styles/_editor.less

```diff
@@ -56,6 +56,11 @@
     .form-horizontal .control-label {
       text-align: left;
     }
+
+    .deprecated {
```

Comment:
Nesting selectors creates higher selector specificity and implies a very tight semantic coupling between the elements that use the selectors. Since this class can be applied to any text, not just just within `.visualization-options`, I think we should rename it `.text-color-deprecated` and move it into [base.less](https://github.com/elastic/kibana/blob/master/src/ui/public/styles/base.less). There are some similar-looking classes in that file, so I think it fits.